### PR TITLE
Force ongoing or recurring never both fix

### PR DESCRIPTION
**Pull request by srikanthmaturu**
Added two radio buttons single day and multi day to select the duration type. This is showin in create event and edit event pages (Only exception is when the user edits an already existing eventdatetime that is recurring and multi day).

Updated the date-time select box to enable/disable recurring options when the user switches between single day and multi day duration types.

### CODE REVIEW

File: www/templates/default/html/manager/AddDatetime.tpl.php [hunk 1]
```diff
@@ -130,74 +135,92 @@
         <label for="room">Room</label>
         <input type="text" id="room" name="room" value="<?php echo $datetime->room; ?>" />
 
+        <?php #Don't show the box when the current eventdatetime is a recurring instance or current evendatetime is a recurring and multi-day type.
+            if ($context->recurrence_id == NULL && !($datetime->recurringtype != 'none' && $event_duration_type == 'multi_day')) : ?>
+            <label for="single-day-event"><span class="required">*</span> Event duration type</label><br>
```
Comment (mfairchild365) [inline www/templates/default/html/manager/AddDatetime.tpl.php]: This doesn't make sense as a label anymore, as it is labeling this group of radio buttons and not an individual radio button. Would using a `<fieldset>` and a `<lengend>` make more sense and be more semantic here?

Example:
```
<fieldset>
<legend>Event duration type</legend>
<label><input type="radio" ...> Single Day </label>
<label><input type="radio" ...> Multi Day </label>
</fieldset>
```

File: www/templates/default/html/manager/AddDatetime.tpl.php [hunk 2]
```diff
@@ -130,74 +135,92 @@
...
+            <input <?php if ($event_duration_type =='single_day') echo 'checked=checked' ?> id="single-day-event" type="radio" value="single_day" name="event_duration_type">
+            <label>Single Day</label>&nbsp;
```
Comment (mfairchild365) [inline www/templates/default/html/manager/AddDatetime.tpl.php]: Labels must be programmatically associated with controls for accessibility. This is the only way the label will be announced by assistive technology such as screen readers.

Options include:
* reference the control via the `id` attribute (explicit association)
* wrap the control in the <label> (implicit association)

File: www/templates/default/html/manager/CreateEvent.tpl.php [hunk 3]
```diff
@@ -60,128 +60,145 @@
...
-                        <option <?php if ($post['start_time_minute'] == 55) echo 'selected="selected"'; ?> value="55">55</option>
-                    </select>
-
-                    <div id="start-time-am-pm" class="am_pm">
-                        <fieldset>
-                            <legend class="wdn-text-hidden">AM/PM</legend>
-                            <label><input <?php if (!isset($post) || $post['start_time_am_pm'] == 'am') echo 'checked="checked"'; ?> id="start-time-am-pm-am" title="AM" type="radio" value="am" name="start_time_am_pm">AM</label><br>
-                            <label><input <?php if ($post['start_time_am_pm'] == 'pm') echo 'checked="checked"'; ?> id="start-time-am-pm-pm" type="radio" value="pm" name="start_time_am_pm">PM</label>
-                        </fieldset>
+                <label for="event-duration-type"><span class="required">*</span> Event duration type</label>
```
Comment (mfairchild365) [inline www/templates/default/html/manager/CreateEvent.tpl.php]: Same label vs fieldset/legend comments as I had in `AddDatetime.tpl.php`.

**Comment by srikanthmaturu** [comment]
Thanks @mfairchild365 for suggestions. I have now used the fieldset approach to group radio buttons. After that my old css was not working anymore, I removed it and used jquery for what I needed to do. Any other issues?

File: www/templates/default/html/manager/AddDatetime.tpl.php [hunk 4]
```diff
@@ -137,23 +137,25 @@
 
         <?php #Don't show the box when the current eventdatetime is a recurring instance or current evendatetime is a recurring and multi-day type.
             if ($context->recurrence_id == NULL && !($datetime->recurringtype != 'none' && $event_duration_type == 'multi_day')) : ?>
-            <label for="single-day-event"><span class="required">*</span> Event duration type</label><br>
-            <input <?php if ($event_duration_type =='single_day') echo 'checked=checked' ?> id="single-day-event" type="radio" value="single_day" name="event_duration_type">
-            <label>Single Day</label>&nbsp;
-            <input <?php if ($event_duration_type == 'multi_day') echo 'checked=checked' ?> id="multi-day-event" type="radio" value="multi_day" name="event_duration_type">
-            <label>Multi Day</label>
+                <fieldset>
+                    <legend style="margin:auto; font-size:.802rem;"><span class="required">*</span> Event duration type</legend>
```
Comment (mfairchild365) [inline www/templates/default/html/manager/AddDatetime.tpl.php]: Why the inline styles?

**Comment by srikanthmaturu** [comment]
Moved the inline css to manager.less. @mfairchild365 any other improvements?

**Comment by jsturek** [comment]
Closing due to the age of PR

**Pull request closed without merging**